coralproject/cay
# Added translations and real language switch

## What does this PR do?

It adds some spanish translations but more importantly it saves/retrieves the language from/to localstorage so it persist during refresh

## How do I test this PR?

- Change language
- See how it persist during sessions

@coralproject/frontend
